code/modules/reagents/reagent_containers/cups/glassbottle.dm

```diff
@@ -579,13 +579,17 @@
 			isGlass = FALSE
 	return
 
+/obj/item/reagent_containers/cup/glass/bottle/molotov/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..(hit_atom, throwingdatum, do_splash = FALSE)
```

Comment:
```suggestion
	..(hit_atom, throwingdatum, do_splash = FALSE)
```
We don't need ``.`` for anything here. Not a big deal but it just bugs me.

Reply:
Respectable, applied